cost/google/schedule_instance/README.md

```diff
@@ -6,30 +6,33 @@ This policy schedules Google VM instances to start and stop at specific times ba
 
 ## How To Use
 
-This policy uses the schedule label value (default key: schedule) for scheduling the instance. The appropriate value should be added to as a label to every instance you want to manage via this policy.
+This policy uses the schedule label value (default key: `schedule`) for scheduling the instance. The appropriate value should be added to as a label to every instance you want to manage via this policy.
 
-This value is a string consisting of 3 underscore-separated substrings:
+### Example Schedule Labels
 
-- *Hours* - Start and stop hours are in 4-digit 24-hour format without any colons or other separators. For example, a value of `0815-1730` will start instances at 8:15 and stop them at 17:30 (5:30 pm). If the minute field is left blank, the minute value of `00` will be assumed.
-- *Days of the Week* - Hyphen-separated list of days indicated by their two-letter abbreviation value from the following list: su,mo,tu,we,th,fr,sa. For example, a value of `mo-tu-we-th-fr` will start and stop the instances on weekdays but not on weekends.
-- *Timezone* - Timezone in [tz database format](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) with the `/` character replaced with a hyphen and all characters converted to lowercase. For example, a value of `america-new_york` would specify US Eastern Time. Defaults to UTC if no Timezone field is provided.
+ - `schedule` = `0000-1200_sa-sun`
+   Start at 12am (midnight) and stop at 12pm (noon) on Saturday and Sunday (UTC timezone)
 
-**Example Value:** 0815-1730_mo-tu-we-th-fr_america-new_york
 
-- Starts instances at 8:15am
-- Stops instance at 5:30pm
-- Monday - Friday, US Eastern Time.
+ - `schedule` = `0815-1730_mo-tu-we-th-fr_america-new_york`
+   Start at 8:15am and stop at 5:30pm every weekday in US Eastern Time (America/New York)
 
-In the above example, instances are off during the weekend and start back up on Monday morning at 8:15am and are off at 5:30pm every weekday. Times are UTC unless the Timezone field is provided.
+### Schedule Label Format
+`<Schedule Label>` = `<Hours>_<Days of the Week>[_Optional Timezone]`
+
+The Schedule Label value is a string consisting of 2 or 3 underscore-separated (`_`) substrings (Hours, Days of the Week, and optional Timezone) with the following format:
```

Comment:
@bryankaraffa how come this one uses underscores and not semicolons? Is it because Google labels do not support semicolons?

Reply:
Correct -- all lowercase, underscore only, etc..